### ISSUE TITLE
Full Accessibility & 508 Office Audit VAOS-Vaccine Distribution

### ISSUE THREAD

**Issue by laurenernest** [department-of-veterans-affairs/va.gov-team#26412]
## Steps to complete the Full Accessibility & 508 Audit: 
- [x] VFS: Verify all accessibility defects that were identified at Staging Review have been fixed before requesting a full audit *(Platform understands that other teams may be accountable for fixing the defects identified at Staging Review. These defects are exempt and will not block a team from scheduling this touchpoint)*
- [x] VFS Product Manager: Create this issue and fill in your team and feature name in the title
- [x] VFS: Using the Slack Workflow ([User guide](https://depo-platform-documentation.scrollhelp.site/support/Getting-help-from-the-Platform-in-Slack.1439138197.html)), notify Collaboration Cycle team of this Github ticket in #vfs-platform-support. 
- [ ] Platform: Create sub-issues attached to this epic with findings. Audits are performed on a first-come-first-serve basis and are scheduled with each biweekly sprint
- [ ] VFS: Coordinate with the VA 508 Office to address accessibility findings
- [ ] Platform: Update [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=1116695886)
- [ ] VFS: Close this ticket when all steps above are completed.

## Artifacts
- Access information for the tool or feature in staging. Do not put staging credentials in your `va.gov-team` ticket; store or reference them in a .md file in the `va.gov-team-sensitive` repository
  1. Go to https://staging.va.gov/health-care/schedule-view-va-appointments/appointments
  2. Log in with [Judy](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-vaos.md)
  3. Select the Start scheduling button
  4. From the Type of Care page, select COVID-19 vaccine
  5. This starts the vaccine flow
- Link to verification that accessibility checks are complete:
  - (preferred) Link to your [TestRail Platform accessibility test plan template](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc).
OR
  - Link to a doc in your product folder that indicates all required checks from the accessibility [staging review prep](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md) doc are complete.
  - #21342
  - #24137
- List of known issues (with links to tickets when applicable)

## Additional information
As part of this review, Platform will send your product to VA's 508 office for approval (as applicable).

Please refer to the [Platform Collaboration Cycle README](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/README.md) on Github for more information about the Collaboration Cycle.

**Comment by noahgelman** [comment]
@laurenernest Upon reviewing the vaccine flow, I have found no accessibility issues.

**Comment by laurenernest** [comment]
Thanks! I sent the email to Linsey for review.

**Comment by laurenernest** [comment]
Sent follow-up email to Linsey to check on this one

**Comment by laurenernest** [comment]
Sent follow-up email to Cristin to check on status

**Comment by laurenernest** [comment]
SCTASK7069573 has been created in ServiceNow

**Comment by JRRoof** [comment]
Opened a VFS Platform Support request and chatted with @briandeconinck.  He confirmed there were no issues found and this process is no longer followed, so closing out this ticket.
